src/xcode/ENA/ENA/Source/Extensions/FileManager+ClearTemporaryDirectory.swift

```diff
@@ -0,0 +1,22 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+ extension FileManager {
+	func removePDFsFromTemporaryDirectory() {
+		do {
+			let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+			let temporaryDirectory = try FileManager.default.contentsOfDirectory(atPath: temporaryDirectoryURL.path)
+			try temporaryDirectory.forEach { file in
+				if file.contains(".pdf") {
+					let fileUrl = temporaryDirectoryURL.appendingPathComponent(file)
+					try FileManager.default.removeItem(atPath: fileUrl.path)
```

Comment:
Do you think we should write a unit test with a dummy file for it, for the BSI I mean?